rust/src/normalization.rs

```diff
@@ -276,10 +276,7 @@ mod bdb_normalize {
         OffDup = 4,    // Off-page duplicate
     }
 
-    lazy_static! {
-        static ref PROC_SELF_CWD: PathBuf = PathBuf::from("/proc/self/cwd");
-        static ref PROC_SELF_FD: PathBuf = PathBuf::from("/proc/self/fd");
```

Comment:
Side benefit - dead code detection now flagged the latter as unused, which I guess it couldn't in the macro form.